metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -542,6 +543,48 @@ object Messages {
     }
   }
 
+  object UpdateScalafmtConf {
+
+    def letUpdate = new MessageActionItem("Let Metals update .scalafmt.conf")
+    def createMessage(
+        sourceItems: List[RelativePath],
+        dialect: ScalafmtDialect
+    ): String = {
+      val directories = sourceItems.mkString("[", ",\n", "]")
+      s"The following source directories can't be formatted by scalafmt " +
+        s"because they require `runner.dialect = ${dialect.value}` setting:" +
```

Comment:
```suggestion
        s"because they require the `runner.dialect = ${dialect.value}` setting:" +
```

Small nit that I actually missed last time.

Reply:
Thx!